backend/src/test/java/codezap/tag/repository/TagJpaRepositoryTest.java

```diff
@@ -0,0 +1,133 @@
+package codezap.tag.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import codezap.global.exception.CodeZapException;
+import codezap.global.repository.JpaRepositoryTest;
+import codezap.tag.domain.Tag;
+
+@JpaRepositoryTest
+class TagJpaRepositoryTest {
+
+    private final TagRepository tagRepository;
+
+    @Autowired
+    TagJpaRepositoryTest(TagRepository tagRepository) {
+        this.tagRepository = tagRepository;
+    }
+
+    @Nested
+    @DisplayName("id로 태그 조회")
+    class fetchById {
+        @Test
+        @DisplayName("성공 : id로 태그를 알아낼 수 있다.")
```

Comment:
```suggestion
        @DisplayName("성공 : id로 태그를 조회할 수 있다.")
```
[제안]
아주 사소하지만, 클래스 단위의 `@DisplayName` 과 용어를 통일시켜 보아도 좋을 것 같아요.